infra/tpu-pytorch-releases/dev_images.auto.tfvars

```diff
@@ -13,5 +13,10 @@ dev_images = [
     accelerator  = "cuda"
     cuda_version = "12.1"
     extra_tags   = ["cuda"]
+  },
+  {
+    accelerator  = "cuda"
```

Comment:
Can you set the python version for at least this image to 3.10? 3.8  (default) hits EOL this year.

Reply:
Done. Thanks for pointing out.